src/commands/farm.rs

```diff
@@ -26,6 +26,14 @@ pub(crate) async fn farm(is_verbose: bool) -> Result<()> {
     install_tracing(is_verbose);
     color_eyre::install()?;
 
+    let instance = SingleInstance::new("subspaceFarmer")
```

Comment:
should we have a constant for FARMER_INSTANCE_NAME or something along those lines? There are a couple of instances  of this.

Reply:
yes, thank you